frontend/packages/dev-console/integration-tests/support/commands/app.ts

```diff
@@ -63,7 +63,7 @@ Cypress.Commands.add('mouseHover', (selector: string) => {
   cy.get(selector)
     .invoke('show')
     .should('be.visible')
-    .trigger('mouseover');
+    .trigger('mouseover', { force: true });
```

Comment:
Seems to be a common usage, left the comment but it's not a big deal.

Reply:
`mouseover` is a globally registered command, but errors out in some cases where an element is being covered by another element so to avoid that, adding force flag to ensure it always triggers a mouseover event on the given target.